### DISCUSSION TITLE
Developer meeting (April 10th): The Crazyflie client

### DISCUSSION THREAD

**Discussion by ataffanel** [bitcraze/discussions#1250] [Announcements]
The next developer meeting will be on the 10th of April 2024, we will talk about the Crazyflie client. It will be based on the content of the [latest blogpost](https://www.bitcraze.io/2024/03/crazyflie-client-past-present-and-future/).

Date: Wednesday, April 10th, 2024
Time: 3 PM Central European Summer Time (check your [timezone](https://www.worldtimebuddy.com/?qm=1&lid=2692969,5368361,1850147&h=2692969&date=2024-04-10&sln=15-16&hf=0))
Topic: Crazyflie client
Hosts: @ataffanel 
Meeting link: Meeting is over! Recording can be found over here: https://youtu.be/QJb79YbMMEs

15:00 - 15:30: Presentation and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

**Notice**: Due to unforeseen circumstances, the dev meeting has been rescheduled the 10th of April.

**Comment by knmcguire** [comment]
The recording can be found here : https://youtu.be/QJb79YbMMEs
